### DISCUSSION TITLE
Sponsorship?

### DISCUSSION THREAD

**Discussion by conoro** [matamorphosis/Scrummage#43] [Q&A] [answered]
Hi, is the project interested in ongoing sponsorship? Would love to discuss. Thanks.

**Comment by matamorphosis** [comment, marked as answer]
Hi @conoro,

I'd love to discuss ongoing sponsorship with you in a more private setting, is there an easy way the project can get in contact with you?

Best regards,

matamorphosis

**Comment by conoro** [comment]
Great yes, thanks! coneill@tines.io
